### ISSUE TITLE
Role stops if firewalld is not installed

### ISSUE THREAD

**Issue by rhmk** [linux-system-roles/sap-preconfigure#48]
When using this role on a system without firewalld installed, the systemctl module throws an error.
So either we add firewalld to the list of required packages or check that it is not installed

**Comment by rhmk** [comment]
Proper service handling can be achieved with the following code snippets:
```

- name: populate service facts
  service_facts:
- name: some_other_task
  some_module:
     some_param: XXX
  when: ansible_facts.services["firewalld.service"] is defined

```
the module service_facts creates an array with allow started and enabled services. This is a way to avoid errors if a service is not even installed (like numad or kdump)

**Comment by berndfinger** [comment]
Thanks for this!
I just approved and merged pull request #49 which solves the issue.
